### PR TITLE
Add copy to clipboard icon

### PR DESCRIPTION
**Pull request by survivant**
# Description
I added the copy to clipboard feature

image

image

### Contribution Checklist:

- [X] **The pull request only addresses one issue or adds one feature.**
- [X] **The pull request does not introduce any breaking changes**
- [X] **I have added screenshots or gifs to help explain the change if applicable.**
- [X] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [X] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

### CODE REVIEW

File: packages/bruno-app/src/components/Sidebar/Collections/Collection/CollectionItem/GenerateCodeItem/CodeView/index.js [hunk 1]
```diff
@@ -20,13 +24,20 @@ const CodeView = ({ language, item }) => {
...
-      readOnly
-      value={snippet}
-      font={get(preferences, 'font.codeFont', 'default')}
-      theme={storedTheme}
-      mode={lang}
-    />
+    <>
+      <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+        <CopyToClipboard text={snippet} onCopy={() => toast.success('Copied to clipboard!')}>
+          <FontAwesomeIcon icon={faCopy} style={{ cursor: 'pointer' }} />
```
Comment (martinsefcik) [inline packages/bruno-app/src/components/Sidebar/Collections/Collection/CollectionItem/GenerateCodeItem/CodeView/index.js]: Based on `contributing.md` file Bruno is using Tabler icons, so probably should be replaced with Tabler icon to not mix icons from different collections.

Comment (survivant) [reply packages/bruno-app/src/components/Sidebar/Collections/Collection/CollectionItem/GenerateCodeItem/CodeView/index.js]: agree

**Comment by martinsefcik** [comment]
What do you think about the following styling? 


https://github.com/usebruno/bruno/assets/1073299/b12f0f5a-c777-458a-b76d-abb9cefdcd1c

**Comment by survivant** [comment]
I'm not really a react dev, so I have know idea how to do it.  It's true that I never changed the layout theme.

**Comment by martinsefcik** [comment]
I just slightly modified `CodeView` component to:
```js
    <StyledWrapper>
      <CopyToClipboard className="copy-to-clipboard" text={snippet} onCopy={() => toast.success('Copied to clipboard!')}>
        <IconCopy size={20} strokeWidth={1.5}/>
      </CopyToClipboard>
      <CodeEditor
        readOnly
        value={snippet}
        font={get(preferences, 'font.codeFont', 'default')}
        theme={storedTheme}
        mode={lang}
      />
    </StyledWrapper>
```
introduced new StyledWrapper for that component:
```js
import styled from 'styled-components';

const StyledWrapper = styled.div`
  position: relative;

  .copy-to-clipboard {
    position: absolute;
    cursor: pointer;
    top: 10px;
    right: 10px;
    z-index: 10;
    opacity: 0.5;

    &:hover {
      opacity: 1;
    }
  }
`;

export default StyledWrapper;
```

**Comment by survivant** [comment]
@martinsefcik Thanks a lot for the help.  It's really appreciated.

**Review by martinsefcik — APPROVED** [review]

**Comment by helloanoop** [comment]
Thank you so much @survivant @martinsefcik !

**Pull request merged**